include/emds/emds.h

```diff
@@ -98,12 +98,14 @@ int emds_init(emds_store_cb_t cb);
  * is called, takes the given data pointer and stores the data to the emergency
  * data storage.
  *
+ * @note emds does not do the local copy of the dynamic entry structure.
```

Comment:
This means that the user of emds need to keep its dynamic entry, as that it what the entry in the list is pointing to. That what was I tried to avoid with this code. So the user could just add this and then forget about this functionality, Now the user has to keep the emds_dynamic_entry in its area.

Reply:
Hm. What is the reason to not keep it on the user side? It doesn't reduce memory usage or something similar. If a user provides it as global or static memory emds guarantees the absence of the issue 'no memory' during dynamic entry allocation or preconfigured memory in advance.
If you use dynamic memory the code depends on the heap fragmentation (Is there any heap automatic defragmentation in Zephyr?) 
If you use dynamic memory in a nondynamic manner you cut off pieces of heap that might be usable by others working in parallel functionalities.

Reply:
We then have to add the emds_dynamic_entry to every server instance that uses emds. That was what I tried to avoid.

Reply:
Allocation of dynamic entry will happen in any case. In the initial case, it will be allocated from the heap within emds. In the current case, it will be static variable inside a model structure. 
We have to have this memory busy in any case but the variable inside the model is the more explicit case.

Reply:
Ok, but then we should remove the CONFIG_HEAP_MEM_POOL_SIZE from the tests prj.conf file, as it is not needed.